assets/sass/_base.sass

```diff
@@ -244,8 +245,7 @@ header
       background-color: white
 
 #mainNav
```

Comment:
Also drop this line?

Reply:
@sftim this one should remain as further attributes belong to the content within `#mainNav` nav element.

Reply:
@sftim this one should remain as further attributes belong to the content within `#mainNav` nav element.

Reply:
Oh, it's Sass and I hadn't noticed. D'oh!